### DISCUSSION TITLE
[DropdownMenu] SubMenu is closing on its own after refecting data using redux (RTK query)

### DISCUSSION THREAD

**Discussion by minedun6** [radix-ui/primitives#2321] [Help]
Hi,
I'd like to get some help with this weird issue.
I'm using cmdk component and the dropdown menu primitive.

https://github.com/radix-ui/primitives/assets/4510058/06c3539b-9bb6-4bea-8515-f38cc61d76a9

I have no idea why only the submenu is getting closed.

**Comment by joaom00** [comment]
It's hard to say without the code. Can you provide a sandbox?

**Reply by minedun6** [reply]
I'll try to setup one as soon as possible
